bin/update_or_create_webhooks.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import json
+
+import click
+from openedx_webhooks.lib.github.utils import update_or_create_webhook
+
+click.disable_unicode_literals_warning = True
+
+
+@click.command()
+@click.argument('config-json', type=click.File('rb'))
+@click.option('--dry-run', is_flag=True)
+def cli(config_json, dry_run):
+    """
+    Update or create the specified webhook into repos using a
+    configuration JSON file:
+
+    \b
+    {
+      "repos": [
+        "owner1/repo1",
+        ...
+      ],
+      "hooks": [
+        {
+          "config": {
+            "url": "payload_url",
+            "insecure_ssl": false, # Optional
+            "secret": "secret", # Optional
+            "content_type": "json" # Optional
+          },
+          "events": [
+            "pull_request",
+            ...
+          ]
+        },
+        ...
+      ]
+    }
+
+    See <https://developer.github.com/v3/repos/hooks/#create-a-hook> for
+    more information about the `config`
+    """
+    raw = json.load(config_json)
+    repos = raw['repos']
+    hook_confs = raw['hooks']
+
+    if dry_run:
+        click.echo(
+            '**Dry run only** The following actions would have been performed:'
+        )
+
+    for repo, conf in ((r, c) for r in repos for c in hook_confs):
```

Comment:
This would be better as a double loop instead of a for statement over a double generator expression.